libs/database/src/lib/entities/localization-keys.entity.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Model,
+  PrimaryKey,
+  Table,
+  Unique
+} from "sequelize-typescript";
+import { BIGINT, NUMBER, STRING } from "sequelize";
+
+
+
+@Table({ tableName: "localization_keys", underscored: true, paranoid: false })
+export class LocalizationKeys extends Model<LocalizationKeys> {
+
+  @PrimaryKey
+  @AutoIncrement
+  @Column(BIGINT.UNSIGNED)
+  override id: number;
+
+  @AllowNull
+  @Column(STRING)
+  key: string | null;
+
+  @AllowNull
+  @Column(STRING)
```

Comment:
This column appears to be a TEXT type, not STRING in the DB (`text` vs `varchar(255)`)